gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/api/ApiReactorHandlerFactory.java

```diff
@@ -157,7 +158,11 @@ public ReactorHandler create(Api api) {
                     apiComponentProvider
                 );
 
-                if (api.getExecutionMode() == null || api.getExecutionMode() == ExecutionMode.V3) {
+                if (
+                    !configuration.getProperty(API_JUPITER_MODE_ENABLED, Boolean.class) ||
```

Comment:
Add default value to false to avoid possible NPE: `!configuration.getProperty(API_JUPITER_MODE_ENABLED, Boolean.class, false)`